src/components/note/note.component.tsx

```diff
@@ -67,10 +88,19 @@ const Note = ({
         )}
 
         <StyledNoteContent>
-          <Editor readOnly editorState={getDecoratedValue(noteContent)} />
+          <Editor
+            readOnly
+            editorState={getDecoratedValue(noteContent)}
+            onChange={() => {}}
+          />
         </StyledNoteContent>
         {React.Children.map(previews, (preview) =>
-          React.cloneElement(preview, { as: "a", onClose: undefined })
+          React.isValidElement(preview) && preview.type === LinkPreview
```

Comment:
I think we should use `displayName` here just in case consumers pass the `LinkPreview`s in their own wrapper component